packages/core/src/types/IImage.ts

```diff
@@ -120,6 +120,12 @@ interface IImage {
 
   imageQualityStatus?: ImageQualityStatus;
   calibration?: IImageCalibration;
+  imageFrame?: any;
+
+  bufferView?: {
+    buffer: ArrayBuffer;
+    offset?: number;
```

Comment:
Make the offset required if the bufferView is defined.
Consider including the string representation of the view as well (eg 'Uint8array')